src/keys-builder/utils/get-current-translation.ts

```diff
@@ -0,0 +1,43 @@
+import * as fsExtra from 'fs-extra';
+import { po } from 'gettext-parser';
+import { unflatten } from 'flat';
+
+import { getConfig } from '../../config';
+
+function getJson(path) {
+  return fsExtra.readJsonSync(path, { throws: false }) || {};
+}
+
+function getPot(path) {
```

Comment:
They are under an object called `parser`, they should be called: `parseJson` & `parsePot`